apps/explorer/tsconfig.json

```diff
@@ -20,7 +20,8 @@
       "@cowprotocol/abis": ["../../../libs/abis/src/index.ts"],
       "@cowprotocol/analytics": ["../../../libs/analytics/src/index.ts"],
       "@cowprotocol/assets": ["../../../libs/assets/src/index.ts"],
-      "@cowprotocol/types": ["../../../libs/types/src/index.ts"]
+      "@cowprotocol/types": ["../../../libs/types/src/index.ts"],
+      "@cowprotocol/wallet": ["../../../libs/wallet/src/index.ts"]
```

Comment:
Should we import the wallet into the Explorer?
Probably shouldn't be needed for analytics.

Reply:
Yes, I rather remove it please